lib/class/class_notification_service.dart

```diff
@@ -285,51 +137,136 @@ class NotificationService {
+        description:
+            localizations.warning_severity_explanation_dialog_minor_description,
+        groupId: "de.nucleus.foss_warn.notification_group",
+        importance: Importance.max,
+      ),
+      AndroidNotificationChannel(
+        "de.nucleus.foss_warn.notifications_moderate",
+        localizations.notification_settings_notify_by_moderate,
+        description: localizations
+            .warning_severity_explanation_dialog_moderate_description,
+        groupId: "de.nucleus.foss_warn.notification_group",
+        importance: Importance.max,
+      ),
+      AndroidNotificationChannel(
+        "de.nucleus.foss_warn.notifications_severe",
+        localizations.notification_settings_notify_by_severe,
+        description:
+            localizations.warning_severity_explanation_dialog_minor_description,
+        groupId: "de.nucleus.foss_warn.notification_group",
+        importance: Importance.max,
+      ),
+      AndroidNotificationChannel(
+        "de.nucleus.foss_warn.notifications_extreme",
+        localizations.notification_settings_notify_by_extreme,
+        description: localizations
+            .warning_severity_explanation_dialog_extreme_description,
+        groupId: "de.nucleus.foss_warn.notification_group",
+        importance: Importance.max,
+      ),
+      AndroidNotificationChannel(
+        "de.nucleus.foss_warn.notifications_update",
+        localizations.notification_settings_notify_by_update,
+        description: localizations
+            .warning_severity_explanation_dialog_update_description,
+        groupId: "de.nucleus.foss_warn.notification_group",
+        importance: Importance.low,
+      ),
+      AndroidNotificationChannel(
+        "de.nucleus.foss_warn.notifications_other",
+        localizations.notification_channel_other_name,
+        description: localizations.notification_channel_other_description,
+        groupId: "de.nucleus.foss_warn.notification_group",
+        importance: Importance.defaultImportance,
+      ),
+    ];
+
+    final androidNotificationPlugin =
+        _flutterLocalNotificationsPlugin.resolvePlatformSpecificImplementation<
+            AndroidFlutterLocalNotificationsPlugin>();
+    if (androidNotificationPlugin != null) {
+      // init the different notifications channels/groups
+      try {
+        // create notification group for grouping multiple notifications
+        await androidNotificationPlugin.createNotificationChannelGroup(
+          AndroidNotificationChannelGroup(
+            "de.nucleus.foss_warn.notification_group",
+            localizations.notification_group_name,
+            description: localizations.notification_group_description,
+          ),
+        );
+
+        // create all notification channels
+        for (var channel in notificationChannels) {
+          await androidNotificationPlugin.createNotificationChannel(
+            channel,
+          );
+        }
+      } catch (e) {
```

Comment:
Please don't do these generic catches. Figure out what exact errors are being thrown and catch them accordingly. We need to get a 1.0.0 out so I guess it's ok for now but afterwards we'll have to get rid of these kind of things.